src/lib.rs

```diff
@@ -120,7 +120,7 @@ pub use crate::reference::{Reference, ReferenceNames, References};
 pub use crate::reflog::{Reflog, ReflogEntry, ReflogIter};
 pub use crate::refspec::Refspec;
 pub use crate::remote::{
-    FetchOptions, PushOptions, Refspecs, Remote, RemoteConnection, RemoteHead,
+    FetchOptions, PushOptions, Refspecs, Remote, RemoteConnection, RemoteHead, RemoteRedirect,
```

Comment:
API question: `RemoteRedirect` or `Redirect`? The libgit2 enum is called `git_remote_redirect_t`.